src/odemis/gui/model/__init__.py

```diff
@@ -509,6 +517,10 @@ def __init__(self, main):
         # It is set at start-up by the tab controller, and will never be active.
         self.semStream = None
 
+        self.roa = model.TupleContinuous((0, 0, 1, 1),
```

Comment:
Why do you initialise this roa with full FoV by default? The SPARC used to start with the ROA undefined, and it should stay like this.